action.yml

```diff
@@ -1,28 +1,29 @@
 # See: https://docs.github.com/en/actions/creating-actions
-name: TODO:Action Name
+name: Gem Push
 author: FreeAgent
-description: TODO:Description
-#  inputs:
-#    foo:
-#      description: Foo level
-#      required: true
-#      default: 23
-#  outputs:
-#    bar:
-#      description: Bar output
+description: Push gem packages to a rubygems compatible repo
+inputs:
+  package-glob:
+    description: "File glob to match the .gem files to push"
+    default: "pkg/*.gem"
 
 runs:
-  # Pick one of the 3 possible types of action:
-
-  #using: 'composite'
-  #steps:
-  #  - name: Step One
-  #    shell: bash
-  #    run: |
-  #      echo TODO: Step One
-
-  #using: 'node12'
-  #main: main.js
-
-  #using: 'docker'
-  #image: 'Dockerfile'
+  using: "composite"
+  steps:
+    - name: Push Gem
+      shell: bash
+      env:
+        # Expects GEM_HOST and GEM_HOST_API_KEY to be set
+        GEM_GLOB: ${{ inputs.package-glob }}
```

Comment:
why not use the `${{ input.package-glob }}` in the `run`?

Reply:
Because when testing I often pull the run code into a file and actually run it. At some point when it gets much bigger, that is how it will sit in the repo as well, this will then all still work. The `${{ }}` confuses bash and bash linters when not expanded! I did actually prototype that way and then pull it out.

Also, annoyingly for Docker and JS actions they automatically set `INPUT_FOO` for all the inputs, but not for composite actions where it would be really useful.